src/Symfony/Component/Routing/Attribute/Route.php

```diff
@@ -71,6 +75,7 @@ public function __construct(
         $this->setMethods($methods);
         $this->setSchemes($schemes);
         $this->setAliases($alias);
+        $this->setEnvs((array) $env);
```

Comment:
When `$env = null`, this should be an empty array, not `[null]`. Please add a unit test for this.
```suggestion
        $this->setEnvs($env ? (array) $env : []);
```

Reply:
I think this might not be necessary 🤔
In PHP, casting null to an array — (array) null — results in an empty array ([]).
I tested this from PHP 8.1 to 8.4, and the behavior is consistent across all versions.
Also, I added some tests to cover this case.

Resources:

`script.php`
```php
<?php

$var = null;

var_dump((array) $var);
```

**results**
<img width="1297" height="534" alt="Screenshot from 2025-08-09 10-25-26" src="https://github.com/user-attachments/assets/62b123ee-35a0-46e0-b90e-08924eae64cf" />

Reply:
Thank you for checking.